src/common/Client.js

```diff
@@ -83,6 +83,23 @@ class Client {
             .then((r) => r && r.data);
     }
 
+    postBlob(url, formData, params, headers = {}) {
+        console.log(url);
```

Comment:
Whoops. This should probably be removed :)

Reply:
yep!